app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(peo["name"], peo["age"]) for peo in people]
+    for hus in people:
+        if "wife" in hus and hus.get("wife"):
+            Person.people.get(hus.get("name")).wife = \
+                Person.people[hus.get("wife")]
```

Comment:
```suggestion
            Person.people[hus["name"]].wife = Person.people[hus["wife"]]
```
In this case not need .get because we already know that wife exist in hus
Also we know that we already create hus with name
The same fix with husband

Reply:
Because .get can return None or you must provide default value, but we no have default value in this case
https://docs.quantifiedcode.com/python-anti-patterns/correctness/not_using_get_to_return_a_default_value_from_a_dictionary.html?highlight=get